### PR TITLE
Fix Bug 1133145 - Update meta tags on 'Check your plugins' page

### PR DESCRIPTION
**Pull request by kyoshino**
There are 2 new strings behind the `meta_update_201403` tag. Cc @flodolo

### CODE REVIEW

**Comment by kyoshino** [comment]
There are 2 new strings. Cc @flodolo

File: bedrock/mozorg/templates/mozorg/plugincheck.html [hunk 1]
```diff
@@ -4,9 +4,25 @@
 
 {% extends "base-resp.html" %}
 
-{% block body_id %}plugincheck{% endblock %}
+{% block page_image %}{{ static('img/plugincheck/page-image.png') }}{% endblock %}
+
+{% block page_title %}
+  {% if l10n_has_tag('meta_update_201403') %}
```
Comment (flodolo) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: Same here, 201503

Comment (kyoshino) [inline bedrock/mozorg/templates/mozorg/plugincheck.html]: Thanks @flodolo :blush: Fixed!

**Comment by alexgibson** [comment]
This one is an r+ from me too. Will wait to notify l10n that we are good to merge this

**Comment by flodolo** [comment]
Good to merge.

**Pull request merged**